openrtb/impression.go

```diff
@@ -17,7 +17,7 @@ type Impression struct {
 
 	// No TagId(tagid).
 
-	BidFloorPrice    float64  `json:"bidfloor"`
+	BidFloorPrice    *float64 `json:"bidfloor"`
```

Comment:
Do we need to encode the `BidFloorPrice` as null when the object is `nil` explicitly? Otherwise, maybe `json:"bidfloor,omitempty"`